gui/src/installer/step/descriptor.rs

```diff
@@ -989,17 +992,29 @@ impl DescriptorEditModal for EditXpubModal {
                     }
                 }
             },
+            Message::UpgradeLedger(id, network) => {
```

Comment:
message could be instead a Message::HardwareWallets(hws::HardwareWalletMessage::UpgradeLedger(id))
and then would be handle not here but in the upper level in the hws.update

Reply:
I handle the message here (and in other modals) as i need to access to `EditXpubModal.upgrading` to disable th `on_blur()` action to edit the modal.

Moving the handling to `hws.update()` will centralized the handling but could add some complexity no notify each modal of the `upgrading` state change(some modals in App and some in Installer).